api/app/createNewGameVsAi.js

```diff
@@ -0,0 +1,20 @@
+const { newGameVsAi } = require("../game");
+const { save, updatePlayer1 } = require("../database/game");
+const playAiMove = require("./playAiMove");
+const { CROSS_VALUE, CIRCLE_VALUE } = require("../constants/game");
+
+const getRandomTeam = () =>
+  Math.floor(Math.random() * Math.floor(2)) === 0 ? CROSS_VALUE : CIRCLE_VALUE;
+
+const createNewGameVsAi = async () => {
+  const game = newGameVsAi();
+  const newGame = await save(game);
+  const iaTeam = getRandomTeam();
+  await updatePlayer1(newGame.id, iaTeam);
+  if (newGame.currentPlayer === iaTeam) {
+    playAiMove(newGame);
```

Comment:
`playAiMove` doesn't return a new game ?

Reply:
No, I don't need it. The playAiMove will update the game in DB and it will be display when the app refresh the game.